### PR TITLE
[WIP] set optimizeForSpeed to true to showcase issue

### PR DESCRIPTION
**Pull request by rgbkrk**
This PR is up merely to show an issue we're currently having with `styled-jsx`. If you run `npm run app:commuter` on this branch, open your devtools, and navigate between pages with the directory listing, you'll see the following warnings in the console:

screen shot 2018-08-08 at 12 14 20 pm

Navigate again and the whole page will break down with errors:

screen shot 2018-08-08 at 12 15 08 pm

In summary, it looks like Chrome is rejecting stylesheets that are being added (saying they're invalid) and then later styled-jsx appears to be trying to remove styles that are not there (due to never getting added properly).

This is styled-jsx 2.2.6 and affects both next 5 and next 6.

### CODE REVIEW

**Comment by rgbkrk** [comment]
Reported this in styled jsx as well: https://github.com/zeit/styled-jsx/issues/474

**Comment by rgbkrk** [comment]
This was only a demonstration of a problem. We're now upgraded anyway and styled-jsx is going after the fix so I'm going to just close this PR.
